os/arch/arm/src/imxrt/chip/imxrt_usb.h

```diff
@@ -0,0 +1,757 @@
+#define IMXRT_USBOTG_HCOR_BASE               (IMXRT_USB_BASE+IMXRT_USBOTG_HCOR_OFFSET)
+#define IMXRT_USBOTG_USBCMD                  (IMXRT_USB_BASE+IMXRT_USBOTG_USBCMD_OFFSET)
+#define IMXRT_USBOTG_USBSTS                  (IMXRT_USB_BASE+IMXRT_USBOTG_USBSTS_OFFSET)
+#define IMXRT_USBOTG_USBINTR                 (IMXRT_USB_BASE+IMXRT_USBOTG_USBINTR_OFFSET)
+#define IMXRT_USBOTG_FRINDEX                 (IMXRT_USB_BASE+IMXRT_USBOTG_FRINDEX_OFFSET)
+#define IMXRT_USBHOST_PERIODICLIST           (IMXRT_USB_BASE+IMXRT_USBHOST_PERIODICLIST_OFFSET)
+#define IMXRT_USBDEV_DEVICEADDR              (IMXRT_USB_BASE+IMXRT_USBDEV_DEVICEADDR_OFFSET)
+#define IMXRT_USBHOST_ASYNCLISTADDR          (IMXRT_USB_BASE+IMXRT_USBHOST_ASYNCLISTADDR_OFFSET)
+#define IMXRT_USBDEV_ENDPOINTLIST            (IMXRT_USB_BASE+IMXRT_USBDEV_ENDPOINTLIST_OFFSET)
+#define IMXRT_USBHOST_TTCTRL                 (IMXRT_USB_BASE+IMXRT_USBHOST_TTCTRL_OFFSET)
+#define IMXRT_USBOTG_BURSTSIZE               (IMXRT_USB_BASE+IMXRT_USBOTG_BURSTSIZE_OFFSET)
+#define IMXRT_USBHOST_TXFILLTUNING           (IMXRT_USB_BASE+IMXRT_USBHOST_TXFILLTUNING_OFFSET)
+#define IMXRT_USBOTG_BINTERVAL               (IMXRT_USB_BASE+IMXRT_USBOTG_BINTERVAL_OFFSET)
+#define IMXRT_USBDEV_ENDPTNAK                (IMXRT_USB_BASE+IMXRT_USBDEV_ENDPTNAK_OFFSET)
+#define IMXRT_USBDEV_ENDPTNAKEN              (IMXRT_USB_BASE+IMXRT_USBDEV_ENDPTNAKEN_OFFSET)
+#define IMXRT_USBOTG_PORTSC1                 (IMXRT_USB_BASE+IMXRT_USBOTG_PORTSC1_OFFSET)
+#define IMXRT_USBOTG_OTGSC                   (IMXRT_USB_BASE+IMXRT_USBOTG_OTGSC_OFFSET)
+#define IMXRT_USBOTG_USBMODE                 (IMXRT_USB_BASE+IMXRT_USBOTG_USBMODE_OFFSET)
+
+/* Device side naming of common register offsets */
+
+#define IMXRT_USBDEV_USBCMD                  IMXRT_USBOTG_USBCMD
+#define IMXRT_USBDEV_USBSTS                  IMXRT_USBOTG_USBSTS
+#define IMXRT_USBDEV_USBINTR                 IMXRT_USBOTG_USBINTR
+#define IMXRT_USBDEV_FRINDEX                 IMXRT_USBOTG_FRINDEX
+#define IMXRT_USBDEV_BURSTSIZE               IMXRT_USBOTG_BURSTSIZE
+#define IMXRT_USBDEV_BINTERVAL               IMXRT_USBOTG_BINTERVAL
+#define IMXRT_USBDEV_PORTSC1                 IMXRT_USBOTG_OTGSC
+#define IMXRT_USBDEV_USBMODE                 IMXRT_USBOTG_USBMODE
+
+/* Host side naming of common registers */
+
+#define IMXRT_USBHOST_USBCMD                 IMXRT_USBOTG_USBCMD
+#define IMXRT_USBHOST_USBSTS                 IMXRT_USBOTG_USBSTS
+#define IMXRT_USBHOST_USBINTR                IMXRT_USBOTG_USBINTR
+#define IMXRT_USBHOST_FRINDEX                IMXRT_USBOTG_FRINDEX
+#define IMXRT_USBHOST_BURSTSIZE              IMXRT_USBOTG_BURSTSIZE
+#define IMXRT_USBHOST_BINTERVAL              IMXRT_USBOTG_BINTERVAL
+#define IMXRT_USBHOST_PORTSC1                IMXRT_USBOTG_OTGSC
+#define IMXRT_USBHOST_USBMODE                IMXRT_USBOTG_USBMODE
+
+/* Device endpoint registers */
+
+#define IMXRT_USBDEV_ENDPTSETUPSTAT          (IMXRT_USB_BASE+IMXRT_USBDEV_ENDPTSETUPSTAT_OFFSET)
```

Comment:
same as above and please check this in entire file.

Reply:
I fixed it